### PR TITLE
preload profile image

### PR DESCRIPTION
**Pull request by JoshDeiner**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,7 +41,7 @@
 				<div class="container">
 					<div class="row">
 						<div class="col-4 col-5-large col-12-medium">
-							<span class="image fit"><img src="images/chris-diep.webp" alt="Christopher Diep or Chris Diep headshot" /></span>
+							<span class="image fit"><link rel="preload" as="image" alt="Christopher Diep or Chris Diep headshot" href="images/chris-diep.webp"></span>
```
Comment (christphrd) [inline index.html]: Still unsure what this does. (TESTING)

**Review by christphrd — DISMISSED** [review]

**Comment by christphrd** [comment]
Not sure how to un-approve this

**Comment by JoshDeiner** [comment]
it is basically an instruction to the browser's preload scanner to fetch this image early on https://web.dev/articles/optimize-lcp

**Comment by christphrd** [comment]
> it is basically an instruction to the browser's preload scanner to fetch this image early on https://web.dev/articles/optimize-lcp

I don't see the image and see this error in my browser's console: 
```
The resource file:///Users/christopherdiep/development/static-personal-website/images/chris-diep.webp was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
```

**Comment by JoshDeiner** [comment]
> > it is basically an instruction to the browser's preload scanner to fetch this image early on https://web.dev/articles/optimize-lcp
> 
> I don't see the image and see this error in my browser's console:
> 
> ```
> The resource file:///Users/christopherdiep/development/static-personal-website/images/chris-diep.webp was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
> ```

close the pr. this a warning not an error. read these docs. to increase your score you'll probably need to make some non quick fixes. 

https://engeniusweb.com/why-your-mobile-site-speed-score-is-bad-and-why-thats-ok/

https://web.dev/articles/optimize-lcp

**Pull request closed without merging**